src/elements/common/async.js

```diff
@@ -0,0 +1,3 @@
+const importV4Annotations = () => import(/* webpackChunkName: "box-annotations" */ 'box-annotations');
```

Comment:
Does this work for the pre-built bundles and self-hosted assets? We use code splitting like this elsewhere in the app, but we add the `webpackMode: "lazy"` directive, which is [replaced conditionally|]https://github.com/box/box-ui-elements/blob/master/scripts/webpack.config.js#L159 based on which asset we're building. We likely need to add it here, too, though it will have an impact on load performance in those use-cases.

Reply:
I added the `webpackMode` and set it to `lazy` and retested with the HOC and it is working via pre-built.  Though, I just tried using the `dist/preview.js` file directly and it doesnt seem to load box-annotations into the global object.  Do you have any suggestions on this?

Reply:
It's probably a matter of fussing with the webpack config so that it creates `dist/box-annotations.js`. Does it need to be added to the `entries` map in webpack.config.js?